backend/app/services/notifications/utils/content.py

```
@@ -0,0 +1,47 @@
+
+def notification_type_for_status(status: str) -> str:
+    if status == "failed":
+        return NotificationType.EVAL_FAILED.value
+    return NotificationType.EVAL_COMPLETED.value
+
+
+def format_completed_at(dt: datetime | None) -> str:
+    if not dt:
+        return ""
+    if dt.tzinfo is None:
+        dt = dt.replace(tzinfo=timezone.utc)
```

Ruff (0.15.18)

[warning] Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)